src/tests/ftest/util/mpio_utils.py

```diff
@@ -69,15 +66,18 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
                 romio_test_repo --built romio test directory
         """
 
-        # environment variables only to be set on client node
-        env_variables = ["export CRT_ATTACH_INFO_PATH={}/install/tmp/" \
-                             .format(basepath),
-                         "export MPI_LIB=''", "export DAOS_SINGLETON_CLI=1"]
-
-        # setting attributes
-        cd_cmd = 'cd ' + romio_test_repo
-        test_cmd = './runtests -fname=daos:test1 -subset -daos'
-        run_cmd = cd_cmd + ' && ' + test_cmd
+        env = EnvironmentVariables()
+        env["CRT_ATTACH_INFO_PATH"] = "{}/install/tmp/".format(basepath)
+        env["DAOS_SINGLETON_CLI"] = 1
+        env["D_LOG_FILE"] = os.environ.get("D_LOG_FILE", "")
+        env["OFI_INTERFACE"] = os.environ.get("OFI_INTERFACE", "eth0")
```

Comment:
Specific reason this is "eth0"? As we have been trying to move over to 'ib0', will be nice to have the same here.

Reply:
`eth0` should be the _reasonable_ default as it's ubiquitous.  Any environment that wants to use `ib0` should set that environment variable in the caller (as our CI does) and this code will pick that up.